<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineInput_username">
     <property name="placeholderText">
      <string>Insert username</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_activateUser">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_login">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineInput_password">
     <property name="placeholderText">
      <string>Insert Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_createUser">
     <property name="text">
      <string>New User</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
